### PR TITLE
Add support to ROS2 Galactic

### PR DESCRIPTION
**Pull request by TSC21**
This pull request adds support for using this tool with ROS2 Galactic.

### CODE REVIEW

**Comment by TSC21** [comment]
Run the `tox` tests with success locally:
```sh
---------------------------------------------------------------------------------------------------------------------- live log call -----------------------------------------------------------------------------------------------------------------------
INFO     ros_cross_compile.sysroot_creator:sysroot_creator.py:126 Building sysroot image: nuno/aarch64-ubuntu-eloquent:latest
INFO     ros_cross_compile.sysroot_creator:sysroot_creator.py:135 Successfully created sysroot docker image: nuno/aarch64-ubuntu-eloquent:latest
PASSED                                                                                                                                                                                                                                               [ 98%]
test/test_sysroot_creator.py::test_create_sysroot_stage_creation PASSED                                                                                                                                                                              [100%]

----------- coverage: platform linux, python 3.8.5-final-0 -----------
Coverage XML written to file coverage.xml


============================================================================================================== 52 passed in 717.57s (0:11:57) ==============================================================================================================
_________________________________________________________________________________________________________________________ summary __________________________________________________________________________________________________________________________
  py: commands succeeded
  congratulations :)
```

**Comment by christophebedard** [comment]
This looks good to me, but I'd defer to someone who knows more about cross_compile. I approved the CI run and it's looking good though! The shellcheck linter failure is not now.

**Comment by TSC21** [comment]
> This looks good to me, but I'd defer to someone who knows more about cross_compile. I approved the CI run and it's looking good though! The shellcheck linter failure is not new.

@christophebedard thanks! I guess we wait on @emersonknapp?

**Comment by christophebedard** [comment]
> @christophebedard thanks! I guess we wait on @emersonknapp?

yeah, unless someone else can review.

File: test/test_entrypoint.py
Comment (emersonknapp) [inline test/test_entrypoint.py:49]: This doesn't seem necessary to change - the test against foxy is still valid (and will stay valid for a year longer)

File: README.md
Comment (emersonknapp) [inline README.md:274]: Given that Foxy will outlive Galactic - I would prefer to leave the current LTS ROS 2 Distro as the example in the README. Would you mind reverting that portion of this?

**Review by emersonknapp — COMMENTED** [review]
Quick nitpicks to address - looks good after that. Thanks for this

File: test/test_entrypoint.py
Comment (TSC21) [reply test/test_entrypoint.py:49]: yeah that's ok. Jut thought we want to keep testing the latest stable.

File: README.md
Comment (TSC21) [reply README.md:274]: Sure all good.

**Comment by TSC21** [comment]
@emersonknapp should be good now. Can you issue a new release in PyPi so this is made available by `pip`?

**Comment by TSC21** [comment]
CI doesn't seem to be moving for some reason

**Review by emersonknapp — APPROVED** [review]

**Comment by emersonknapp** [comment]
Please be patient - you've added a few dimensions to the test matrix - and the queue is currently backed up. Once it passes I'll merge this, and we can cut a pip release after that. We can expect to have the new version on PyPI by EOD tomorrow.

**Comment by TSC21** [comment]
> Please be patient - you've added a few dimensions to the test matrix - and the queue is currently backed up. Once it passes I'll merge this, and we can cut a pip release after that. We can expect to have the new version on PyPI by EOD tomorrow.

Well I do believe that it's just Github Actions queueing for now reason. Before it was much quicker to process them. Thanks for for this and for merging it!

**Comment by emersonknapp** [comment]
The release is out https://pypi.org/project/ros-cross-compile/0.7.0/

**Comment by TSC21** [comment]
> The release is out https://pypi.org/project/ros-cross-compile/0.7.0/

@emersonknapp thanks!

**Pull request merged**
